deepmd/entrypoints/train.py

```diff
@@ -68,11 +71,15 @@ def train(
 
     jdata = normalize(jdata)
 
-    jdata = update_sel(jdata)
+    if is_compress == False:
+        jdata = update_sel(jdata)
 
     with open(output, "w") as fp:
         json.dump(jdata, fp, indent=4)
 
+    # save the training script into the graph
+    tf.constant(json.dumps(jdata), name='training_script', dtype=tf.string)
```

Comment:
Do we need to consider to use `zlib.compress` to save the compressed string? This can reduce the model size.

Reply:
TensorFlow will automatically compress the tensors within the frozen model, so I don’t think we need to manually compress the json script.